en/lessons/specifics/benchmarking.md

```diff
@@ -0,0 +1,346 @@
+* **formatters** - a list of formatter functions you'd like to run to output the benchmarking results of the suite when using `Benchee.run/2`. Functions need to accept one argument (which is the benchmarking suite with all data) and then use that to produce output. Defaults to the builtin console formatter calling `Benchee.Formatters.Console.output/1`. We'll cover this more in a later section.
+* **print** - a map with the following options as atoms for the keys and values of either `true` or `false`. This lets us control if the output identified by the atom will be printed during the standard benchmarking process. All options are enabled by default (true). Options are:
+  * **benchmarking** - print when Benchee starts benchmarking a new job.
+  * **configuration** - a summary of configured benchmarking options including estimated total run time is printed before benchmarking starts.
+  * **fast_warning** - warnings are displayed if functions are executed too fast, potentially leading to inaccurate measures.
+* **console** - a map with the following options as atoms for the keys and variable values. The available values are listed under each option:
+  * **comparison** - if the comparison of the different benchmarking jobs (x times slower than) is shown. Defaults to `true`, but can also be set to `false`.
+  * **unit_scaling** - the strategy for choosing a unit for durations and counts. When scaling a value, Benchee finds the "best fit" unit (the largest unit for which the result is at least 1). For example, `1_200_000` scales to 1.2 M, while `800_000` scales to 800 K. The unit scaling strategy determines how Benchee chooses the best fit unit for an entire list of values, when the individual values in the list may have different best fit units. There are four strategies, all given as atoms, defaulting to `:best`:
+    * **best** - the most frequent best fit unit will be used. A tie will result in the larger unit being selected.
+    * **largest** - the largest best fit unit will be used
+    * **smallest** - the smallest best fit unit will be used
+    * **none** - no unit scaling will occur. Durations will be displayed in microseconds, and ips counts will be displayed without units.
+
+## Inputs
+
+It's very important to benchmark your functions with data that reflects what that function might actually operate on in the real world. Frequently a function can have very different behavior on small sets of data versus large sets of data! This is where Benchee's `inputs` configuration option comes in. This allows you to test the same function but with as many different functions as you like, and then you can see the results of the benchmark with each of those functions.
```

Comment:
"a function can have very different behavior" I think I'd say "a very different..." but you're the native speaker :)

Reply:
Here you often say "different functions" while you mean different inputs I believe :)